.circleci/config.yml

```diff
@@ -43,32 +52,68 @@ jobs:
           paths: [ ~/.m2/repository ]
           key: mac-java-{{ .Branch }}-{{ checksum "pom.xml" }}
 
-  sonar_analysis:
-    executor: jdk11-docker
-    # A large machine is needed to perform the whole analysis, as it requires a lot of memory
+  launch_compatibility_checks:
```

Comment:
maybe find a better name e.g. download_sandboxes_artifacts or something similar

Reply:
that's not what this job does, it's just a blocking step in case the _**run_compatibility**_ parameter is set to false